subgraphs/compound-forks/protocols/aurigami/src/mapping.ts

```diff
@@ -302,11 +323,138 @@ function getOrCreateProtocol(): LendingProtocol {
     "Aurigami",
     "aurigami",
     "2.0.1",
-    "1.1.6",
+    "1.1.7",
     "1.0.0",
     Network.AURORA,
     comptroller.try_liquidationIncentiveMantissa(),
     comptroller.try_oracle()
   );
   return _getOrCreateProtocol(protocolData);
 }
+
+/////////////////
+//// Helpers ////
+/////////////////
+
+function getPrice(
+  marketAddress: Address,
+  priceOracle: string
+): ethereum.CallResult<BigInt> {
+  //
+  //
+  // There are 4 markets where the price oracle does not work
+  // PLY, AURORA, TRI, USN
+  // Use Trisolaris LP pool pairs to derive price
```

Comment:
You could remove this comment by rewriting the `if`'s below in a self descriptive way like:
```
if (marketUnuspportedByOracle(marketAddress)) {
  return getMarketPriceFromLp(marketAddress);
}
```

Then remove boilerplate like:
```
const customOracle = constants.CustomOracles[marketAddress];
return getPriceFromLP(priceOracle, customOracle.KnownMarket, marketAddress, customOracle.LP);
```

Reply:
Wouldn't I still need to have a list of if statements? In order to get the proper LP pair contract address?

Reply:
Well, yeah. Either that or put the constants in a map that allows you to fetch them directly. But yeah 😅

Reply:
I am going to leave as is for now, but that is a good idea